src/mbgl/renderer/painters/painter_raster.cpp

```diff
@@ -42,7 +42,8 @@ static std::array<float, 3> spinWeights(float spin) {
 void Painter::renderRaster(PaintParameters& parameters,
                            RasterBucket& bucket,
                            const RenderRasterLayer& layer,
-                           const RenderTile& tile) {
+                           const mat4& matrix,
+                           bool useBucketBuffers = false) {
```

Comment:
@kkaefer - The painter methods are layer specific as opposed to source specific. Adding a separate painter function would also impact the RasterBucket which calls this method. 
Changing the method signature here woks for all RasterBuckets and RasterLayers, as opposed to creating a cascade of additional changes. 
It also allows us to reuse this painter to render raster layers with other non-tiled sources.